src/script/components/RichTextEditor/plugins/TypeaheadMenuPlugin.tsx

```diff
@@ -350,18 +351,28 @@ function LexicalPopoverMenu<TOption extends TypeaheadOption>({
 }): JSX.Element | null {
   const [menuVisible, setMenuVisible] = useState(false);
   const [selectedIndex, setHighlightedIndex] = useState<null | number>(null);
+  const defaultSelectedIndex = isReversed ? options.length - 1 : 0;
 
   const anchorElementRef = useMenuAnchorRef({
     containerId,
     resolution: resolution,
     setResolution,
     className: `typeahead-menu ${anchorClassName || ''}`,
     menuVisible,
+    onAdded: () => {
+      scrollToOption(defaultSelectedIndex, options, containerId);
+    },
   });
 
   useEffect(() => {
-    setHighlightedIndex(isReversed ? options.length - 1 : 0);
-  }, [options.length, isReversed]);
+    setHighlightedIndex(defaultSelectedIndex);
+  }, [defaultSelectedIndex]);
+
+  useEffect(() => {
+    if (selectedIndex) {
+      scrollToOption(selectedIndex, options, containerId);
+    }
+  }, [containerId, options, selectedIndex]);
```

Comment:
we now react to a change of the selected index to trigger a scroll to the element selected (it's more explicit now)